models/intel/bert-large-uncased-whole-word-masking-squad-embedding-0001/accuracy-check.yml

```
@@ -0,0 +1,35 @@
+models:
+  - name: bert-large-uncased-whole-word-masking-squad-embedding-0001
+    launchers:
+      - framework: dlsdk
+        allow_reshape_input: True
+        adapter:
+            type: bert_question_answering_embedding
+            embedding: "embedding"
```

Model has no 'embedding' output

```suggestion
            embedding: "3157/Fused_Add_"
```